themes/api.jquery.com/index.php

```diff
@@ -2,6 +2,10 @@
 <div class="content-right listing twelve columns">
 	<div id="content">
 		<h1 class="page-title">jQuery API</h1>
+		<p>jQuery is a fast, small, and feature-rich JavaScript library. It makes things like HTML document traversal and manipulation, event handling, animation, and Ajax much simpler with an easy-to-use API that works across a multitude of browsers. If you're new to jQuery, you might be interested in the <a href="http://learn.jquery.com/">jQuery Learning Center</a> tutorials.</p>
+
+		<p>The API docs aren't specifically versioned, but you need to pay attention to the version info on each page.</p>
```

Comment:
The first time your visiting the api docs this makes sense IMHO. I'm going to update my PR later to include some other stuff discussed in https://github.com/jquery/api.jquery.com/issues/44 too.